### PR TITLE
Add docs on how to use reserved words in KSQL.

### PR DESCRIPTION
**Pull request by hjafarpour**
### Description 
Add docs on how to handle columns with reserved words.

### Testing done 
_Describe the testing stategy. Unit and integration tests are expected for any behavior changes._

### Reviewer checklist
- [ ] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ ] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

File: docs/syntax-reference.rst [hunk 1]
```diff
@@ -222,6 +222,20 @@ Example:
 
 .. _create-table:
 
+If the name of a column in your source topic is one of the reserved words in KSQL you can use back quotes to define the column.
+The same applies to the field names in a STRUCT type.
+For indtance, if in the above example we had another field called ``Properties`` which is a reserved word in KSQL you can
```
Comment (JimGalasyn) [inline docs/syntax-reference.rst]: Wee typo: "instance".

File: docs/syntax-reference.rst [hunk 2]
```diff
@@ -478,6 +505,20 @@ Records written into the stream are not timestamp-ordered with respect to other
 Therefore, the topic partitions of the output stream may contain out-of-order records even
 if the source stream for the query is ordered by timestamp.
 
+
+In either of CREATE STREAM AS SELECT, CREATE TABLE AS SELECT or INSERT INTO statements, if you have a column name
+that is a reserved word in KSQL you can access it by using back quotes. Note that when you use back quotes the
+identifier name is case sensitive. The same applies to the field names in STRUCT type.
+In the following example we select a column with name ``Properties`` and a field from address colunm named ``select``:
```
Comment (JimGalasyn) [inline docs/syntax-reference.rst]: Add a comma: "In the following example we, ..."

File: docs/syntax-reference.rst [hunk 3]
```diff
@@ -308,6 +322,19 @@ Example:
 
 .. _create-stream-as-select:
 
+If the name of a column in your source topic is one of the reserved words in KSQL you can use back quotes to define the column.
+The same applies to the field names in a STRUCT type.
+For indtance, if in the above example we had another field called ``Properties`` which is a reserved word in KSQL you can
```
Comment (JimGalasyn) [inline docs/syntax-reference.rst]: Same as above.

File: docs/syntax-reference.rst [hunk 4]
```diff
@@ -478,6 +505,20 @@ Records written into the stream are not timestamp-ordered with respect to other
 Therefore, the topic partitions of the output stream may contain out-of-order records even
 if the source stream for the query is ordered by timestamp.
 
+
+In either of CREATE STREAM AS SELECT, CREATE TABLE AS SELECT or INSERT INTO statements, if you have a column name
```
Comment (JimGalasyn) [inline docs/syntax-reference.rst]: "In any of..."

File: docs/syntax-reference.rst (hunk 1 above)
Comment (hjafarpour) [reply docs/syntax-reference.rst]: Done!

File: docs/syntax-reference.rst (hunk 2 above)
Comment (hjafarpour) [reply docs/syntax-reference.rst]: Comma after `we`?

File: docs/syntax-reference.rst (hunk 4 above)
Comment (hjafarpour) [reply docs/syntax-reference.rst]: Done!

**Pull request merged**